operator/config/istio/kustomization.yaml

```diff
@@ -10,5 +10,14 @@ resources:
 generatorOptions:
   disableNameSuffixHash: true
 
-patchesStrategicMerge:
-  - patches/istio_namespace_labels.yaml
+patches:
```

Comment:
I have an solution in this PR already, not only namespace, but also name should be changed. https://github.com/kyma-project/lifecycle-manager/pull/262/files#diff-6ff3fbcd21a686f5d672eb6d639fb2b9c018e668eb3dd2588589f09465659186R1-R6

Reply:
Name is automatically corrected using `namePrefix`